### PR TITLE
Added greeting from Ethan

### PR DESCRIPTION
**Pull request by ethangraybar**
I added a greeting for myself.

### CODE REVIEW

**Review by alijawosti — DISMISSED** [review]

**Review by TyBeas — DISMISSED** [review]

File: intro-to-git-everyone/user_info/Ethan.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Ethan Graybar
```
Comment (NicMcPhee) [inline intro-to-git-everyone/user_info/Ethan.md]: This file shouldn't be here. I think this is from when you had checked out a copy of the repository inside another copy of the repository. We definitely don't want to commit that. So I would remove this file before merging in this pull request.

**Review by NicMcPhee — CHANGES_REQUESTED** [review]
You really should remove the file `intro-to-git-everyone/user_info/Ethan.md` before you merge this in.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -9,6 +9,7 @@ public class Hellos {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(ethanSaysHello());
     builder.append(andreasSaysHello());
     builder.append(elySaysHello());
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: You've got your alphabetization wrong here, which is why your tests are failing. You want to be after Ely.

```suggestion
    builder.append(andreasSaysHello());
    builder.append(elySaysHello());
    builder.append(ethanSaysHello());
```

**Review by NicMcPhee — CHANGES_REQUESTED** [review]

File: user_info/Ethan.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Ethan Graybar
+
+- I prefer to be called Ethan.
+- My pronouns are he/him.
+- My github username is `ethangraybar`.
+- My github homepage is [github.com/ethangraybar](https://github.com/kklamberty/).
```
Comment (KenBroden) [inline user_info/Ethan.md]: You left in KK's github link here

**Review by NicMcPhee — APPROVED** [review]

**Review by jmahoney5 — APPROVED** [review]
Looks Good

**Pull request merged**